plugins/@grouparoo/facebook/public/templates/destination/{{{id}}}.js.template

```diff
@@ -4,7 +4,7 @@ exports.default = async function buildConfig() {
       id: {{{id}}},
       name: {{{id}}},
       class: "destination",
-      type: "facebook-audiences-custom",
+      type: "facebook-export-audiences-custom",
```

Comment:
The facebook names are a little long, but it seems clear

Reply:
Note - there is not a template for a `facebook-export-audiences-lookalike` @bleonard was that on purpose?

Reply:
yeah, never prioritized building it out all the way.